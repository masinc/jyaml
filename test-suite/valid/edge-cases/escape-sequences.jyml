# Escape sequences in strings
"quotes": "He said \"Hello World\""
"backslash": "Path: C:\\Users\\Name"
"unicode": "Unicode: \u03B1\u03B2\u03B3"
"newlines": "Line 1\nLine 2\tTabbed"