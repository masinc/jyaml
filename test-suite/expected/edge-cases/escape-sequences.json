{"quotes": "He said \"Hello World\"", "backslash": "Path: C:\\Users\\Name", "unicode": "Unicode: αβγ", "newlines": "Line 1\nLine 2\tTabbed"}